client/src/components/share/share-template.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ShareRedirectProps } from './types';
+
+export const ShareTemplate: React.ComponentType<ShareRedirectProps> = ({
+  redirectURL
+}) => {
+  const { t } = useTranslation();
+  return (
+    <a
+      title='Share on Twitter'
```

Comment:
```suggestion
      title={t('buttons.share-on-twitter') || 'Share on Twitter'}
```

Need to use the translation here. Fallback added to appease TypeScript.